govcd/vdccomputepolicy_v2.go

```diff
@@ -0,0 +1,241 @@
+package govcd
```

Comment:
Let me know if this new file is OK or I should simply put everything in vdccomputepolicy.go

Reply:
IMO it is good. Easier deprecation/cleanup in future.